plugins/stats_table_procedure.cc

```diff
@@ -372,11 +376,19 @@ run(const ProcedureRunConfig & run,
 /* STATS TABLE FUNCTION                                                      */
 /*****************************************************************************/
 
+static const std::string INJECT_NOISE_DOC_STR =
+     "Inject laplacian noise to counts. This is useful when training "
+     "a classifier on the examples that were used to genereate the counts. "
```

Comment:
Should we say that this is acceptable only if the count values are relatively large compare to the noise?  How large?